Basic/Basic/MainViewController.m

```diff
@@ -0,0 +1,62 @@
+//
+//  MainViewController.m
+//  Basic
+//
+//  Created by Dima on 07.08.17.
+//  Copyright © 2017 Dima. All rights reserved.
+//
+
+#import "MainViewController.h"
+#import "DetailViewController.h"
+
+@interface MainViewController()
+
+@property(strong, nonatomic) Organization *organization;
+@property(strong, nonatomic) Employee *em;
```

Comment:
`strong` shows ownership over the object. In this case you're just storing a temporary link to the object, so you don't own it. Replace with `weak`. Also, rename it to `selectedEmployee`